CHANGELOG.md

```diff
@@ -34,6 +34,9 @@
     * `BTVenmoDriver.authorizeAccount(profileID:vault:completion:)`
   * Remove `initWithNumber` and `initWithParameters` initializers from `BTCard`
   * Replace `BTVenmoDriver.authorizeAccount` methods with `BTVenmoDriver.tokenizeVenmoAccount`
+  * Remove `BTDataCollectorDelegate`
```

Comment:
Wonder if we should add a subsection for data collector updates? similar to what we did for PP updates in #612

Reply:
Good call - DONE!